examples/line_sender.py

```diff
@@ -0,0 +1,19 @@
+from questdb.ilp import Sender
+
+if __name__ == "__main__":
+    with Sender(host='localhost', port=9009) as sender:
```

Comment:
This example, and more like this, including authentication and TLS, shoud be exposed to the docs.